content/en/docs/tasks/traffic-management/ingress/service-apis/test.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+source "tests/util/samples.sh"
+
+# @setup profile=none
+
+# install Service API CRDs
+snip_setup_1
+
+# install Istio with PILOT_ENABLED_SERVICE_APIS flag enabled
+istioctl install --set values.pilot.env.PILOT_ENABLED_SERVICE_APIS=true -y
+_wait_for_deployment istio-system istiod
+_wait_for_deployment istio-system istio-ingressgateway
+
+startup_httpbin_sample
+
+# setup the Gateway and GatewayClass
+snip_configuring_a_gateway_2
+
+# configure $INGRESS_HOST and $INGRESS_PORT
+_set_ingress_environment_variables
+
+# send CURL traffic to http://$INGRESS_HOST:$INGRESS_PORT/get (expected 200)
+_verify_contains snip_configuring_a_gateway_3 "200 OK"
```

Comment:
I'm thinking this might be more along a 
```
  _verify_like snip_configuring_a_gateway_3 "$snip_configuring_a_gateway_3_out"
 ```
  This way we are also verifying the output shown on the page.

Reply:
Since we leave out a large part of the `curl -sI` output and leave it as `...` in the docs:
```
$ curl -s -I -HHost:httpbin.example.com "http://$INGRESS_HOST:$INGRESS_PORT/status/200"
HTTP/1.1 200 OK
server: istio-envoy
...
```
I'm not sure `_verify_like` would work since it expects same number of lines?

Reply:
`_verify_elided` is another popular verify function, that would probably work here.

Reply:
Awesome, didn't realize that was a thing, changed :+1:

Reply:
FYI, I think all of the currently supported verify functions are listed here: https://github.com/istio/istio.io/blob/master/tests/README.md#verify-functions